README.md

```diff
@@ -145,6 +145,17 @@ Isolator.adapters.sidekiq.ignore_if { Thread.current[:sidekiq_postpone] }
 
 You can add as many _ignores_ as you want, the offense is registered iff all of them return false.
 
+### Using with legacy codebases
```

Comment:
Let's split the instructions into two parts: "Using with Rails" and "Using with Ruby".

We need an example on how to use ignore config in non-Rails apps too.